src/software/ai/hl/stp/play/ball_placement/ball_placement_play_test.py

```diff
@@ -91,6 +103,54 @@ def ball_placement_play_setup(
 )
 def test_two_ai_ball_placement(
     simulated_test_runner, ball_start_point, ball_placement_point
+):
+    run_ball_placement_scenario(
+        simulated_test_runner, ball_start_point, ball_placement_point
+    )
+
+
+@pytest.mark.parametrize(
+    "ball_start_point, ball_placement_point",
+    [
+        # 2023 RoboCup ball placement scenarios
+        # Scenario 1
+        (tbots_cpp.Point(-0.2, -2.8), tbots_cpp.Point(-0.2, 2.8)),
+        # Scenario 2
+        (tbots_cpp.Point(-3.5, -2.25), tbots_cpp.Point(0, 0)),
+        # Scenario 3
+        (tbots_cpp.Point(-1.5, -2.25), tbots_cpp.Point(-0.2, -2.8)),
+        # Scenario 4
+        (tbots_cpp.Point(-4.4, -2.9), tbots_cpp.Point(-0.2, 2.8)),
+        # Scenario 5
+        (tbots_cpp.Point(-0.5, -0), tbots_cpp.Point(-4.3, 2.8)),
+        # Scenario 6
+        (tbots_cpp.Point(-1, -3.15), tbots_cpp.Point(-3.5, -2.8)),
+        # Scenario 7
+        (tbots_cpp.Point(-1, 3.15), tbots_cpp.Point(-3.5, 2.8)),
+        # Scenario 8
+        (tbots_cpp.Point(-4.45, -0.1), tbots_cpp.Point(-0.5, 2.8)),
+    ],
+)
+def test_robocup_technical_challenge_placement(
+    simulated_test_runner, ball_start_point, ball_placement_point
+):
+    run_ball_placement_scenario(
+        simulated_test_runner, ball_start_point, ball_placement_point, blue_only=True
+    )
+
+
+"""
+Runs a ball placement test scenario with the specified parameters.
+
+:param simulated_test_runner: The test runner used to simulate robot and ball behavior.
+:param ball_start_point: The initial position of the ball (provided by pytest parameterization).
+:param ball_placement_point: The target position where the ball should be placed (provided by pytest parameterization).
+:param blue_only: If True, only the blue team is active; the yellow team is ignored.
+"""
+
+
+def run_ball_placement_scenario(
+    simulated_test_runner, ball_start_point, ball_placement_point, blue_only=False
 ):
```

Comment:
Docstring should go under the function declaration

```suggestion
def run_ball_placement_scenario(
    simulated_test_runner, ball_start_point, ball_placement_point, blue_only=False
):
    """
    Runs a ball placement test scenario with the specified parameters.
    
    :param simulated_test_runner: The test runner used to simulate robot and ball behavior.
    :param ball_start_point: The initial position of the ball (provided by pytest parameterization).
    :param ball_placement_point: The target position where the ball should be placed (provided by pytest parameterization).
    :param blue_only: If True, only the blue team is active; the yellow team is ignored.
    """
```